configCivcraft.yml

```diff
@@ -4985,7 +4986,7 @@ recipes:
     output:
       aether:
         material: GOLD_NUGGET
-        amount: 10
+        amount: 3
```

Comment:
The issue here is not balance, but math. If the output is 3, having only 55% production would still leave you with 2 essences. With numbers this small the rounding error gets obscene. Leave pylons unchanged and balance in the xp recipes